pkg/src/src/main/scala/edu/berkeley/cs/amplab/sparkr/SparkRBackendHandler.scala

```diff
@@ -144,6 +144,7 @@ class SparkRBackendHandler(server: SparkRBackend) extends SimpleChannelInboundHa
       args: Array[java.lang.Object],
       parameterTypes: Array[Class[_]]): Boolean = {
     if (parameterTypes.length != numArgs) {
+      // println(s"num of arguments numArgs not match with ${parameterTypes.length}")
```

Comment:
Could we have this line back ? Actually the best thing to do might be to format this similar to the one below saying `Expect parameterTypes.length but got numArgs arguments`